server/apps/publish/subscribers.py

```diff
@@ -123,6 +132,30 @@ def on_create(self, docs):
     def on_update(self, updates, original):
         self._validate_seq_num_settings(updates)
 
+    def _get_subscribers_by_filter_condition(self, filter_condition):
+        req = ParsedRequest()
+        all_subscribers = list(super().get(req=req, lookup=None))
```

Comment:
Perhaps better to leave this as is since it's only used for iterating over it (unless it's not possible to iterate over whatever `get()` returns more than once).